backend/src/router_test.go

```diff
@@ -450,11 +450,23 @@ func (suite *RouterSuite) TestProfile() {
 		suite.JSONEq(`{
 			"user": `+stringify(User{Username: "Username"})+`
 		}`, w.Body.String())
+		
+		w = PATCH("/profile", gin.H{"firstname": "Elk","lastname": "Cloner",}, cookies...)
```

Comment:
We should split this off into a separate top-level functions for each method (`TestGetProfile`/`TestPatchProfile`). Some of the earlier ones do this but I guess these didn't; we can make that change here.

Reply:
I split it up